README.rst

```diff
@@ -48,7 +48,7 @@ The vision is to create a package for finite volume simulation with applications
 
 You are welcome to join our forum and engage with people who use and develop SimPEG at: https://simpeg.discourse.group/.
 
-Weekly meetings are open to all. They are generally held on Wednesdays at 10:30am PDT. Please see the calendar (`GCAL <https://calendar.google.com/calendar/b/0?cid=ZHVhamYzMWlibThycWdkZXM5NTdoYXV2MnNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`_, `ICAL <https://calendar.google.com/calendar/ical/duajf31ibm8rqgdes957hauv2s%40group.calendar.google.com/public/basic.ics>`_) for information on the next meeting.
+Weekly meetings are open to all. They are generally held on Wednesdays at 3:00 PM PDT. Please see the calendar (`GCAL <https://calendar.google.com/calendar/b/0?cid=ZHVhamYzMWlibThycWdkZXM5NTdoYXV2MnNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`_, `ICAL <https://calendar.google.com/calendar/ical/duajf31ibm8rqgdes957hauv2s%40group.calendar.google.com/public/basic.ics>`_) for information on the next meeting.
```

Comment:
I know it's a bit nitpicky, but PDT is specifically Pacific Daylight Time and can't occur during the winter months :). Stupid daylight savings time....

Reply:
You're right! Let's change it to Pacific Time then, which is always in sync with local time in Vancouver.